datasets/ch/seco_sanctions/crawler.py

```diff
@@ -499,28 +498,44 @@ def parse_entry(context: Context, target: Element, programs, places):
                     context, prompt, value, OtherInfo, model=LLM_VERSION
                 )
                 review = request_review(
-                    context,
-                    value,
-                    value,
-                    PLAIN,
-                    "other-information",
-                    None,
-                    extraction,
-                    VERSION,
+                    context=context,
+                    key_parts=value,
+                    source_value=value,
+                    source_mime_type=PLAIN,
+                    source_label="other-information",
+                    source_url=None,
+                    orig_extraction_data=extraction,
+                    model_version=CRAWLER_VERSION,
                 )
 
         if review.accepted:
             for extracted_value in review.extracted_data.simple_values:
-                if "Date" in extracted_value.property:
-                    h.apply_date(
-                        entity, extracted_value.property, extracted_value.value
+                prop = entity.schema.get(extracted_value.property)
+                if prop is not None:
+                    if prop.type == registry.date:
+                        h.apply_date(
```

Comment:
Not for this PR (it would be a bigger thing), but we have this pattern so often that I'm asking myself whether the zavod `Entity` should maybe have a shortcut for this in `.add`?